src/modules/getScores.js

```diff
@@ -0,0 +1,11 @@
+import baseUrl from './baseUrl.js';
+import gameId from './gameId.js';
+
+const getScores = async () => {
+  const url = `${baseUrl + gameId}/scores/`;
+  const res = await fetch(url);
+  const data = await res.json();
+  return data.result;
+};
+
```

Comment:
- [OPTIONAL] It would be nice to display the scores from the highest to the lowest after fetching data from the API endpoint since it is a Leaderboard application. 👍 You can consider using the JavaScript [sort](https://www.freecodecamp.org/news/javascript-array-sort-tutorial-how-to-use-js-sort-methods-with-code-examples/) method to achieve that. 👍 😎